### PR TITLE
PSMDB-1098 Release notes PSMDB 4.4.15-15

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   source/conf.py
	modified:   source/release_notes/index.rst
new file: source/release_notes/4.4.15-15.rst

### CODE REVIEW

**Review by igorsol — APPROVED** [review]
LGTM

**Review by sandraromanchenko — APPROVED** [review]

**Comment by kwilktx** [comment]
Change this

“`SERVER-65821 <https://jira.mongodb.org/browse/SERVER-65821>`_ - Fixed the deadlock on running the `setFCV <https://www.mongodb.com/docs/manual/reference/command/setFeatureCompatibilityVersion/>`_ command when writing the commit decision for the transactions in the "prepared" stage”


To this:

`SERVER-65821 <https://jira.mongodb.org/browse/SERVER-65821>`_ - This fixes deadlock situation that can occur with cross shard transactions and setting the FCV after the “prepared” state that end up with both the setFCV thread and the TransactionCoordinator hung.
`setFCV <https://www.mongodb.com/docs/manual/reference/command/setFeatureCompatibilityVersion/>`

**Comment by kwilktx** [comment]
Change this:

`SERVER-65131 <https://jira.mongodb.org/browse/SERVER-65131>`_ - Fixed uneven distribution of workload over the secondary nodes by disabling opportunistic read targeting - the ability of `mongos` to establish connections to multiple secondaries and direct the request to the first secondary that accepts the connection.

To this:

`SERVER-65131 <https://jira.mongodb.org/browse/SERVER-65131>`_ - This is a v6.0 backport fix to v4.4 that disabled opportunistic read targeting  (except for specified hedged reads) in order to prevent possible performance problems associated with uneven read distribution across the secondaries. 

BACKGROUND: In earlier 4.4 versions the mongos was not evenly distributing secondary reads across all secondaries which caused an overload and bottleneck on the secondary which was consistently resolving 1st. https://jira.mongodb.org/browse/SERVER-57230 
This caused an increase in the number of slow queries when moving from v4.2 to v4.4 - https://jira.mongodb.org/browse/SERVER-57232

**Comment by kwilktx** [comment]
Change this:

 `SERVER-54900 <https://jira.mongodb.org/browse/SERVER-54900>`_ - Fixed an issue with the delay in selecting a new sync-source by canceling the ASIO session when SSL handshake times out

To this:

 `SERVER-54900 <https://jira.mongodb.org/browse/SERVER-54900>`_ - Fixed an issue where competing/blocking network calls to the sync source can prevent selecting a new sync-source. This is resolved by canceling the ASIO session when SSL handshake times out.

**Comment by kwilktx** [comment]
Once the above has been completed, we should be ready to go.

**Comment by nastena1606** [comment]
Amended

File: source/release_notes/4.4.15-15.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Release Highlights
+==================
+
+* Support of :ref:`multiple KMIP servers <kmip>` adds failover to your data-at-rest encryption setup.
+* Allow users to set KMIP client certificate password through a flag to simplify the migration from MongoDB Enterprise to |PSMDB|. 
+
+Other improvements and bug fixes introduced by MongoDB and included in Percona Server for MongoDB are the following:
+
+* `SERVER-66433 <https://jira.mongodb.org/browse/SERVER-66433>`_ - This is a v6.0 backport fix to v4.4 that disables opportunistic read targeting (except for specified hedged reads) in order to prevent possible performance problems associated with uneven read distribution across the secondaries.
```
Comment (sandraromanchenko) [inline source/release_notes/4.4.15-15.rst]: It seems to be a mixup with SERVER-65131, please check

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: source/release_notes/4.4.15-15.rst (hunk 1 above)
Comment (nastena1606) [reply source/release_notes/4.4.15-15.rst]: Thank you! Fixed

**Review by sandraromanchenko — APPROVED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**